SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -98,13 +99,17 @@ bool USpatialNetDriver::InitBase(bool bInitAsClient, FNetworkNotify* InNotify, c
 		bPersistSpatialConnection = true;
 	}
 
+	// Initialize ActorGroupManager as it is a depdency of ClassInfoManager (see below)
+	ActorGroupManager = NewObject<UActorGroupManager>();
+	ActorGroupManager->Init();
```

Comment:
Could ClassInfoManager just own ActorGroupManager internally?

Reply:
This is classic Sahil behaviour just copying my suggestions

Reply:
As discussed offline, ActorGroupManager is intended to become a first class API so decided to get NetDriver to own it.